servers/lib/src/bootstrap.ts

```diff
@@ -0,0 +1,16 @@
+import { NestFactory } from "@nestjs/core";
+import { ConfigService } from "@nestjs/config";
+import * as dotenv from "dotenv"
+import AppModule from "./app.module";
+
+type BootstrapOptions = {
+  config?: string
+}
+
+export default async function bootstrap(options?: BootstrapOptions) {
+  dotenv.config({path: options?.config ?? ".env", override: true});
```

Comment:
There is a subtle bug here.

```
DTaaS/servers/lib (pr-169*) » ls -a                                                                         
.   dist  .eslintignore  jest.config.json  .madgerc       node_modules  README.md  src   tsconfig.json   yarn.lock
..  .env  .eslintrc      LICENSE.md        nest-cli.json  package.json  script     test  yarn-error.log


DTaaS/servers/lib (pr-169*) » ls -a ../.                                                                    .  ..  config  execution  lib


DTaaS/servers/lib (pr-169*) » node dist/src/main.js --config ../.benv                                       [Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [NestFactory] Starting Nest application...
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [InstanceLoader] AppModule dependencies initialized +11ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [InstanceLoader] ConfigHostModule dependencies initialized +0ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [InstanceLoader] ConfigModule dependencies initialized +0ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [InstanceLoader] GraphQLSchemaBuilderModule dependencies initialized +1ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [InstanceLoader] FilesModule dependencies initialized +0ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [InstanceLoader] GraphQLModule dependencies initialized +1ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [GraphQLModule] Mapped {/lib, POST} route +78ms
[Nest] 249290  - 10/17/2023, 3:19:41 PM     LOG [NestApplication] Nest application successfully started +2ms
```

The `.env` of working directory overrides the `-c` flag.